src/index.ts

```diff
@@ -85,7 +85,8 @@ export default class VideoReporter extends WdioReporter {
       return
     }
 
-    this.#outputDir = runner.config.outputDir as string
+    this.#outputDir = this.options.outputDir ?? runner.config.outputDir as string
+    this.#outputDir = this.#outputDir ?? '_results_'
```

Comment:
Let's define the default output dir as constant somewhere at the top.

Reply:
No problem